src/app/shared/components/training-link-panel/training-link-panel.component.html

```diff
@@ -1,11 +1,18 @@
-<div class="govuk-grid-row">
+<div class="govuk-grid-row govuk-!-margin-bottom-5">
   <div *ngIf="canEditEstablishment" class="govuk-grid-column-one-third">
     <a
       class="govuk-list govuk-list--inline govuk-!-margin-right-5"
       [routerLink]="['/workplace', workplace.uid, 'add-mandatory-training']"
       >Manage mandatory training
     </a>
   </div>
+  <div *ngIf="canEditWorker" class="govuk-grid-column-one-third">
+    <a
+      class="govuk-list govuk-list--inline govuk-!-margin-right-5"
+      [routerLink]="['/workplace', workplace.uid, 'change-expires-soon-alerts']"
+      >Change when you get `expires soon` alerts
```

Comment:
```suggestion
      >Change when you get 'expires soon' alerts
```
I would use normal quotation marks here instead of backticks